PERFORMANCE.md

```diff
@@ -4,36 +4,48 @@
 ## Numbers of note
 
 * The barcode is `1024x1024` pixels. The individual tiles are `8x8` in a `9x9` grid (there is an empty row/column of spacing on either side)
-* **7500** or 8750 bytes per cimbar image, after error correction
+* **7500** bytes per cimbar image, after error correction
 	* There are 16 possible symbols per tile, encoding 4 bits
 	* There are 4 or 8 possible colors, encoding an additional 2-3 bits per tile.
-	* These 6-7 bits per tile work out to a maximum of 9300-10850 bytes per barcode, though in practice this number is reduced by error correction.
+	* These 6 bits per tile work out to a maximum of 9300 bytes per barcode, though in practice this number is reduced by error correction.
 	* The default ecc setting is 30/155, which is how we go from 9300 -> 7500 bytes of real data for a 4-color cimbar image.
 		* Reed Solomon is not perfect for this use case -- specifically, it corrects byte errors, and cimbar errors tend to involve 1-3 bits at a time. However, since Reed Solomon implementations are ubiquitous, it is currently in use.
 
 ## Current sustained benchmark
 
-* 4-color cimbar with ecc=30:
+* `mode B` (8x8 4-color) cimbar with ecc=30/155:
+	* 4,689,084 bytes (after compression) in 44s -> 852 kilobits/s (~106 KB/s)
```

Comment:
I'm actually using the same sample file, but the compression level is different. I was encouraged to see mode B holding its own, though I do think it might be slightly more sensitive than the old mode (4C) as far as "best case" performance is concerned.